src/tiledb/cloud/soma/ingest.py

```diff
@@ -402,12 +408,17 @@ def run_ingest_workflow(
         mode=dag.Mode.BATCH,
     )
 
-    # Step 1: Ingest workflow UDF
-    carry_along: Dict[str, str] = {
-        "resources": _DEFAULT_RESOURCES if resources is None else resources,
-        "namespace": namespace,
-        "access_credentials_name": acn,
-    }
+    carry_along: Dict[str, str] = kwargs.pop(
+        "carry_along",
+        {
+            "resources": _DEFAULT_RESOURCES if resources is None else resources,
+            # I don't see why this is needed. I'm not finding any code that
+            # pops "namespace" from kwargs.
+            "namespace": namespace,
```

Comment:
Actually, I do see functions accidentally consuming `namespace`. But let's treat that differently in a different piece of work.